crab/allsamples.py

```diff
@@ -1,648 +1,1071 @@
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_70_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_70_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_70_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_80_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL16MiniAODAPVv2-106X_mcRun2_asymptotic_preVFP_v11-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_80_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_80_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_80_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_90_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL16MiniAODAPVv2-106X_mcRun2_asymptotic_preVFP_v11-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_90_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_90_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2_MINIAODSIM_v0',
+'NMSSM_XYH_Y_tautau_H_gg_MX_900_MY_90_TuneCP5_13TeV-madgraph-pythia8_RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2_MINIAODSIM_v0',
+#TTG/GG
+'TTGG_0Jets_TuneCP5_13TeV-amcatnlo-madspin-pythia8_RunIISummer19UL17MiniAODv2-106X_mc2017_realistic_v9-v1_MINIAODSIM_final',
+'TTGG_0Jets_TuneCP5_13TeV-amcatnlo-madspin-pythia8_RunIISummer19UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v1_MINIAODSIM_final',
+#TTGJets ok
+'TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8_RunIISummer20UL16MiniAODAPVv2-106X_mcRun2_asymptotic_preVFP_v11-v2_MINIAODSIM_final',
+'TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8_RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1_MINIAODSIM_final',
+'TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8_RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v1_MINIAODSIM_final',
+'TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8_RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1_ext1-v1_MINIAODSIM_final',
+# TTJets
+'TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8_RunIISummer19UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v1_MINIAODSIM_final',
+'TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8_RunIISummer20UL16MiniAODAPVv2-106X_mcRun2_asymptotic_preVFP_v11-v1_MINIAODSIM_final',
+'TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8_RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1_MINIAODSIM_final',
+'TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8_RunIISummer20UL17MiniAODv2-106X_mc2017_realistic_v9-v2_MINIAODSIM_final',
+#VBF hgg
+# 16 missing
```

Comment:
Missing because the custom NanoAOD was not produced, or is there another reason, possibly connected to central production?